src/Containers/Calendar/Containers/CalendarTablet/CalendarTablet.story.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+import { storiesOf } from "@storybook/react-native";
+import CalendarTablet from "./CalendarTablet";
+import { View } from "react-native";
+import { action } from "@storybook/addon-actions";
+
+const stories = storiesOf("Calendar|Calendar Tablet", module);
+
+stories.add("Tablet Calendar", () => (
+  <View style={{ width: 650 }}>
```

Comment:
Maybe is better to do it as `maxWidth`? Cause in some rear cases 650 can be more then whole screen.

And to move it to styles. We do not write styles in markup.